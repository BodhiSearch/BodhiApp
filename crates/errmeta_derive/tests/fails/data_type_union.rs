#[derive(errmeta_derive::ErrorMeta)]
union AUnion {
  msg: i32,
  code: u32,
}

fn main() {}
