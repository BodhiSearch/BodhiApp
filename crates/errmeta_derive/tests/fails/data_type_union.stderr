error: proc-macro derive panicked
 --> tests/fails/data_type_union.rs:1:10
  |
1 | #[derive(errmeta_derive::ErrorMeta)]
  |          ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: ErrorMeta can only be derived for enums and structs
